cluster_tools/cluster_tools/schedulers/slurm.py

```diff
@@ -118,6 +118,18 @@ def get_max_array_size():
             )
         return max_array_size
 
+    @staticmethod
+    @cache_in_production
+    def get_max_running():
+        max_running_size_env = os.environ.get("SLURM_MAX_RUNNING", None)
```

Comment:
I'd rename the method to `get_max_running_size` and the env var to `SLURM_MAX_RUNNING_SIZE` (this is also how the debug message mentions it).

Reply:
Cool. No problem; var name changes pushed just now.

Reply:
I also added a sentence to the README, in case you want to accept the pull request directly.